tests/testPupilCoordinates.py

```diff
@@ -162,14 +173,15 @@ def testRaDecFromPupil(self):
                                   rotSkyPos=23.0,
                                   mjd=52000.0)
 
-        nSamples = 100
+        nSamples = 1000
         numpy.random.seed(42)
         ra = (numpy.random.random_sample(nSamples)*0.1-0.2) + numpy.radians(raCenter)
         dec = (numpy.random.random_sample(nSamples)*0.1-0.2) + numpy.radians(decCenter)
         xp, yp = _pupilCoordsFromRaDec(ra, dec, obs_metadata=obs, epoch=2000.0)
         raTest, decTest = _raDecFromPupilCoords(xp, yp, obs_metadata=obs, epoch=2000.0)
-        numpy.testing.assert_array_almost_equal(raTest, ra, decimal=10)
-        numpy.testing.assert_array_almost_equal(decTest, dec, decimal=10)
+        distance = arcsecFromRadians(haversine(ra, dec, raTest, decTest))
+        self.assertLess(distance.max(), 1.0) # because of the imprecision in _icrsFromObserved,
+                                             # the best we can hope for is sub-arcsecond accuracy
```

Comment:
numpy random was generating points too near the Sun.  I changed it so that the test points are all more than 45 degrees from the sun.  Now the accuracy is within 0.005 arcseconds.